### PR TITLE
[logging] rename loggers and set handlers to null

### PR DESCRIPTION
**Pull request by yannmh**
Split `dd.datadogpy` logger to one per module, i.e.
* `datadog.api`
* `datadog.dogstatsd`
* `datadog.threadstats`

Setup the `NullHandler` handler by default as advised in
https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library

### CODE REVIEW

File: datadog/api/resources.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from datadog.api.api_client import APIClient
 
 
-log = logging.getLogger('dd.datadogpy')
+log = logging.getLogger('datadog.api')
```
Comment (degemer) [inline datadog/api/resources.py]: Not used, 🔥  ?

File: datadog/util/config.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 # CONSTANTS
 DATADOG_CONF = "datadog.conf"
 
-log = logging.getLogger('dd.datadogpy')
+log = logging.getLogger('datadog.api')
```
Comment (degemer) [inline datadog/util/config.py]: Not used, 🔥  ?

File: datadog/__init__.py [hunk 3]
```diff
@@ -22,6 +23,10 @@
 
 __version__ = get_version()
 
+# Loggers
+logging.getLogger('datadog.api').addHandler(logging.NullHandler())
+logging.getLogger('datadog.dogstatsd').addHandler(logging.NullHandler())
```
Comment (degemer) [inline datadog/__init__.py]: What about `datadog.threadstats` ?

**Comment by yannmh** [comment]
Fix https://github.com/DataDog/datadogpy/pull/161

**Pull request merged**